pkgs/top-level/all-packages.nix

```diff
@@ -15909,6 +15909,8 @@ with pkgs;
 
   notion = callPackage ../applications/window-managers/notion { };
 
+  ocrmypdf = callPackage ../applications/graphics/ocrmypdf { };
```

Comment:
It is better to use `pythonPackages.callPackage`, and list python dependencies in the arguments of the function that returns the derivation.